docs/pages/next_major_development.md

```diff
@@ -20,6 +20,48 @@ The current tasks are listed here: [Integrate new PMD logo #1931](https://github
 
 The new logo is available from the [Logo Project Page](pmd_projectdocs_logo.html).
 
+### Revamped Command Line Interface
```

Comment:
I hadn't noticed that `7_0_0_release_notes.md`… I'm a little confused as to why we have these separate files. What purpose does each have? should we merge them and keep only one around?

Reply:
As far as I remember: `next_major_development.md` is used to communicate to the community, what we are going to do in PMD 7 (the next major version) - like a preview or "what you can expect". Thus it is in master and should describe the current progress. It is available in all the PMD 6 versions that we release: https://pmd.github.io/pmd-6.50.0/pmd_next_major_development.html

`7_0_0_release_notes.md` are the release notes for PMD 7. We renamed this file from `release_notes.md`, so that we don't overwrite it always when we merge from master->pmd/7.0.x. That's why it is a different file.

For this PR, I'd suggest to move just this new section into `7_0_0_release_notes.md` and leave `next_major_development.md` untouched. May we should remove this file in the pmd/7.0.x branch - since I expect it to be *not* included in the final PMD 7 release.
The only important part are the API changes, that we probably will need to put into the final PMD 7 release notes when the time comes.